usecases/schema/tenant.go

```diff
@@ -212,9 +212,9 @@ func (h *Handler) DeleteTenants(ctx context.Context, principal *models.Principal
 	if err := h.Authorizer.Authorize(principal, authorization.DELETE, authorization.Tenants(class, tenants...)...); err != nil {
 		return err
 	}
-	if err := h.Authorizer.Authorize(principal, authorization.READ, authorization.CollectionsMetadata(class)...); err != nil {
-		return err
-	}
+	// if err := h.Authorizer.Authorize(principal, authorization.READ, authorization.CollectionsMetadata(class)...); err != nil {
+	// 	return err
+	// }
```

Comment:
why is this commented out?

Reply:
this has to be deleted actually

Reply:
i think the comment is in outdated file, because that's already deleted , tenant operation will check for tenant operations